app/src/userpages/components/ProfilePage/IntegrationKeyHandler/IntegrationKeyList/index.jsx

```diff
@@ -1,51 +1,93 @@
 // @flow
 
-import React, { Component } from 'react'
+import React, { useState, useMemo } from 'react'
 import cx from 'classnames'
 
 import type { IntegrationKeyId, IntegrationKey, IntegrationKeyList as IntegrationKeyListType } from '$shared/flowtype/integration-key-types'
 import KeyField from '$userpages/components/KeyField'
-
+import { useBalance, BalanceType } from '$shared/hooks/useBalance'
 import styles from './integrationKeyList.pcss'
 
-export type Props = {
-    integrationKeys: IntegrationKeyListType,
+type CommonProps = {|
```

Comment:
curious, why `|`?

Reply:
Flow wants us to define the type as exact since `item` is added to the props and it thinks it might otherwise conflict.